src/bacnet/basic/object/csv.c

```diff
@@ -90,17 +102,21 @@ void CharacterString_Value_Property_Lists(
  * Initialize the character string values.
  */
 void CharacterString_Value_Init(void)
-{
+{       
```

Comment:
trailing ws

Reply:
you can use `mix format filename` on files that you've edited to correct for format issues.